obj.js

```diff
@@ -214,6 +228,17 @@ async function newObjListener(payload) {
   return createObjectDescription(obj, getNewIDForLocalObject(obj));
 }
 
+async function callListener(payload) {
+  assert(typeof(payload.obj) == "number", "Need object ID");
+  let obj = getLocalObject(payload.obj);
+  let args = mapIncomingObjects(payload.args);
+
+  // may throw
+  let result = obj(...args);
```

Comment:
if `obj` is an object, why is it called as a function?

Is it a constructor? A function?

Reply:
Too much copy and paste, I guess... the local object is actually a Function here, something that the other end wants to `call` (thus this being the `callListener`). Want me to rename it?

Reply:
Yes, please

Reply:
Actually I forgot this is already on `master` now as part of PR#1...